pyproject.toml

```diff
@@ -5,13 +5,14 @@ description = ""
 authors = ["Sasha Weinstein <70385853+SashaWeinstein@users.noreply.github.com>"]
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = ">=3.9,<3.9.7 || >3.9.7,<4.0"
```

Comment:
Why do we have to be this specific with the version of python we are using? I see that in .toml files for other data products we are this specific, just curious if this is automatically updated by poetry or if this is something you had to specify

Reply:
It's a streamlit dependency thing in my understanding. I think the issue is specifically with 3.9.7, this versioning avoids that release